docs/index.md

```diff
@@ -1,11 +1,46 @@
-# What is `earthaccess`?
+![](images/earthaccesslogo.png){alt="hexagonal artwork of a pixalated earth with the word earthaccess" width="10%"}
```

Comment:
Shoot, this syntax doesn't work.
![image](https://github.com/user-attachments/assets/0014c57d-0d81-4d25-91ac-6260de8b5d4b)